src/popup/pages/FungibleTokens/TokenContainer.vue

```diff
@@ -123,16 +123,17 @@ export default defineComponent({
   setup(props, { root }) {
     const { getSdk } = useSdk();
 
+    const tokenType = root.$route.meta?.isCoinDetails ? 'coin' : 'token';
     const contractId = root.$route.params.id;
     const isAe = contractId === AETERNITY_CONTRACT_ID;
     const tabs = [
       {
         text: root.$t('pages.transactionDetails.transactions'),
-        routeName: 'token-transactions',
+        routeName: `${tokenType}-transactions`,
       },
       {
-        text: root.$t(`pages.token-details.${root.$route.params.type}-details`),
-        routeName: 'token-details',
+        text: root.$t(`pages.token-details.${tokenType}-details`),
```

Comment:
As far as I know the string interpolation in terms of the translation paths is considered as a bad practice. Mostly because the i18n scrapers - tools that analyses the rpository and checks which translations are not used. Those tools can't run the JS code, so they are not able to check interpolated paths.
Reference: https://medium.com/doctolib/keys-to-efficient-i18n-ec38fd461d7d#e344